lib/gpx.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+var gpxParse = require("gpx-parse");
+var attributes = require('./attributes'),
+    sportAttr = attributes.sport,
+    trackAttr = attributes.track
+
+var EARTH_RADIUS_KM = 6371.0;
```

Comment:
It's ok for me to not include this file. There might be packages to get the distance of two coordinates.